### ISSUE TITLE
Not display the vorker

### ISSUE THREAD

**Issue by yojikoff** [nicehash/NiceHashMiner-Archived#1175]
The program is launched (two different versions) on one of the machines, but in the private office does not display the vorker.
![1](https://user-images.githubusercontent.com/30227511/28258720-50389ae2-6b05-11e7-82d2-c70464110452.jpg)
The address of the wallet is correct!

**Comment by OneOfMany07** [comment]
Maybe it took a bit of time to update?  Just guessing that often programs aren't immediately able to show all work you just started.

**Comment by yojikoff** [comment]
No! The program has been updated! I watched what was happening for two days ...
